util/top_gen/generator.py

```diff
@@ -365,7 +365,7 @@ def legacy_ordering(input_pin: OutputBlockIo) -> tuple[int, int, int]:
         ),
         ("rtl/templates/sonata_pkg.sv.tpl", "rtl/system/sonata_pkg.sv"),
         ("rtl/templates/pinmux.sv.tpl", "rtl/system/pinmux.sv"),
-        ("doc/ip/pinmux.md.tpl", "doc/ip/pinmux.md"),
+        ("util/templates/pinmux.md.tpl", "doc/ip/pinmux/README.md"),
```

Comment:
I think I would prefer having this in doc/templates/pinmux.md.tpl

Reply:
Leaving this in util for now so that the doc directory contains only readable files.